src/components/SupportRequest.js

```diff
@@ -0,0 +1,236 @@
+          }),
+          h(TextArea, {
+            style: { height: 200, borderTopLeftRadius: 0, borderTopRightRadius: 0, borderTopStyle: 'dashed' },
+            placeholder: 'Enter a description',
+            value: description,
+            onChange: e => this.setState({ description: e.target.value })
+          }),
+          Forms.formLabel('Attachment'),
+          attachmentToken ?
+            div({ style: { display: 'flex', alignItems: 'center' } }, [
+              h(Clickable, {
+                tooltip: 'Change file',
+                style: { flex: 'auto' },
+                onClick: () => this.uploader.current.open()
+              }, [
+                div({
+                  style: { marginLeft: '1rem', paddingTop: '0.5rem' }
+                }, [
+                  'Successfully uploaded: ', span({ style: { color: colors.blue[0] } }, [attachmentName])
+                ])
+              ]),
+              linkButton({
+                tooltip: 'Remove file',
+                style: { flex: 0, paddingTop: '0.5rem' },
+                onClick: () => this.setState({ attachmentToken: '' })
+              }, [icon('times-circle', { size: 23 })])
+            ]) :
+            h(Clickable, {
+              style: {
+                flex: 1, backgroundColor: dragging ? colors.blue[3] : colors.gray[5], borderRadius: 3,
+                border: `1px dashed ${colors.gray[2]}`
+              },
+              onClick: () => this.uploader.current.open()
+            }, [
+              div({ style: { fontSize: 14, lineHeight: '30px', paddingLeft: '1rem' } }, [
+                'Drag or ', link({}, ['Click']), ' to attach a file ',
+                icon('upload-cloud', { size: 25, style: { opacity: 0.4 } })
+              ])
+            ]),
+          uploadingFile && spinnerOverlay,
+          Forms.requiredFormLabel('Contact email'),
+          textInput({
+            value: email,
+            placeholder: 'Enter your email address',
+            onChange: e => this.setState({ email: e.target.value })
+          }),
+          submitError && div({ style: { marginTop: '0.5rem', textAlign: 'right', color: colors.red[0] } }, [submitError]),
+          submitting && spinnerOverlay,
+          div({ style: styles.buttonRow }, [
+            buttonSecondary({
+              style: { marginRight: '1rem' },
+              onClick: () => contactUsActive.set(false)
+            }, ['Cancel']),
+            buttonPrimary({
+              disabled: errors,
+              tooltip: Utils.summarizeErrors(errors),
+              onClick: () => this.submit()
+            }, ['SEND'])
+          ])
+        ])
+      ])
+    ])
+  }
+
+  async submit() {
+    const { onSuccess } = this.props
+    const currUrl = window.location.href
+
+    try {
+      this.setState({ submitting: true })
+      await Ajax().User.createSupportRequest({ ...this.getRequest(), currUrl })
+      onSuccess()
```

Comment:
I got an error when I tried to submit a form: 
![screen shot 2019-01-29 at 1 45 13 pm](https://user-images.githubusercontent.com/2395211/51931989-36ba7280-23cc-11e9-961f-363471580024.png)
The error response in the console says
```
{"request":{"url":"https://broadinstitute.zendesk.com/api/v2/requests/886.json","id":886,"status":"new","priority":"normal","type":null,"subject":"THIS IS A TEST","description":"TESTING new overlay of zendesk modal\n\n------------------\nSubmitted from: http://localhost:3000/#workspaces","organization_id":null,"via":{"channel":"api","source":{"from":{},"to":{},"rel":null}},"custom_fields":[{"id":360012744292,"value":"Kate Voss"},{"id":360012782111,"value":"kateavoss+saturn@gmail.com"},{"id":360012744452,"value":"bug"}],"requester_id":376234121592,"collaborator_ids":[],"email_cc_ids":[],"is_public":true,"due_at":null,"can_be_solved_by_me":false,"created_at":"2019-01-29T18:43:59Z","updated_at":"2019-01-29T18:43:59Z","recipient":null,"followup_source_id":null,"assignee_id":null,"ticket_form_id":360000208511,"fields":[{"id":360012744292,"value":"Kate Voss"},{"id":360012782111,"value":"kateavoss+saturn@gmail.com"},{"id":360012744452,"value":"bug"}]}}
```